_layouts/home.html

```diff
@@ -74,61 +72,6 @@ <h1>“云集”系列栏目</h1>
 
 <div class="strip strip-grey">
   <div class="container pt-6 pb-6 pb-md-10" id="sponsors">
-    <div class="row">
-      <h2>2022 春晚游园赞助商</h2>
-    </div>
-    <div class="row justify-content-start align-items-center">
-      {% for sponsor in site.data.data.sponsors.diamond %}
-      <div class="col-12 col-md-4 mb-1 pl-3 pr-3">
-        <div class="sponsor sponsor-summary">
-          <div class="sponsor-content">
-            {% if sponsor.image %}
-            <div class="sponsor-image">
-              <a href="{{ sponsor.website }}">
-                <img alt="{{ sponsor.name }}" src="{{ sponsor.image }}" />
-              </a>
-            </div>
-            {% endif %}
-            <div class="sponsor-content">
-              {% if sponsor.contact.size > 0 %}
-              {% for contact in sponsor.contact %}
-              {{ contact | markdownify }}
-              {% endfor %}
-              {% endif %}
-            </div>
-          </div>
-        </div>
-      </div>
-      {% endfor %}
-    </div>
-    <div class="row">
-      <h2>2021 秋冬季云集赞助商</h2>
```

Comment:
We can't remove Yunji's sponsors.

Reply:
Yuege will contact the Yunji side at a proper time.

Reply:
sure, added it back.